src/corelibs/U2View/src/ov_msa/MaEditorSequenceArea.cpp

```diff
@@ -101,10 +101,17 @@ MaEditorSequenceArea::MaEditorSequenceArea(MaEditorWgt* ui, GScrollBar* hb, GScr
     addAction(replaceCharacterAction);
     connect(replaceCharacterAction, SIGNAL(triggered()), SLOT(sl_replaceSelectedCharacter()));
 
-    fillWithGapsinsSymAction = new QAction(tr("Fill selection with gaps"), this);
-    fillWithGapsinsSymAction->setObjectName("fill_selection_with_gaps");
-    connect(fillWithGapsinsSymAction, SIGNAL(triggered()), SLOT(sl_fillCurrentSelectionWithGaps()));
-    addAction(fillWithGapsinsSymAction);
+    insertGapsAction = new QAction(tr("Insert gaps"), this);
+    insertGapsAction->setObjectName("insert_gaps");
+    insertGapsAction->setShortcut(QKeySequence(Qt::Key_Space));
+    connect(insertGapsAction, SIGNAL(triggered()), SLOT(sl_insertGaps()));
+    addAction(insertGapsAction);
+
+    replaceWithGapsAction = new QAction(tr("Replace with gaps"), this);
+    replaceWithGapsAction->setObjectName("replace_with_gaps");
+    replaceWithGapsAction->setShortcut(QKeySequence(Qt::CTRL | Qt::Key_Space));
+    connect(replaceWithGapsAction, SIGNAL(triggered()), SLOT(sl_replaceWithGaps()));
```

Comment:
Use functional connectors in new code. Such connectors are checked by the compiler

Reply:
OK